### PR TITLE
Add delay for page reload to fix unstable CreatePortalTest

### PR DESCRIPTION
**Pull request by VeselovAlex**

### CODE REVIEW

File: atsd_webtests/src/test/java/com/axibase/webtest/cases/CreatePortalTest.java [hunk 1]
```diff
@@ -55,7 +58,12 @@ public void createPortal() {
 
         ((JavascriptExecutor) driver).executeScript("document.querySelector('.CodeMirror').CodeMirror.setValue('" + config + "');");
         driver.findElement(By.id("save-button")).click();
-        assertEquals(generateAssertMessage("Title should be 'Portal Test Portal'"), "Portal Test Portal", driver.getTitle());
+        Boolean titleChanged = new FluentWait<>(driver)
+                .withTimeout(3, TimeUnit.SECONDS)
+                .pollingEvery(50, TimeUnit.MILLISECONDS)
+                .withMessage("Title should be 'Portal Test Portal'")
+                .until(ExpectedConditions.titleIs("Portal Test Portal"));
+        assertTrue(generateAssertMessage("Title should be 'Portal Test Portal'"), titleChanged);
```
Comment (annstriganova) [inline atsd_webtests/src/test/java/com/axibase/webtest/cases/CreatePortalTest.java]: What is the difference from changes in [selenide](https://github.com/axibase/atsd-web-test/blob/cd2865d04baba5ed825310c8871897f00c16206b/atsd_webtests/src/test/java/com/axibase/webtest/cases/CreatePortalTest.java#L55) branch?

Comment (VeselovAlex) [reply atsd_webtests/src/test/java/com/axibase/webtest/cases/CreatePortalTest.java]: This issue makes our builds unstable and it should be merged asap. Selenide migration brach is still in development and it doesn't seem to be merged soon

Comment (raipc) [reply atsd_webtests/src/test/java/com/axibase/webtest/cases/CreatePortalTest.java]: The solution doesn't solve the problem as well. Is the failure actually caused by slow page load?

Comment (raipc) [reply atsd_webtests/src/test/java/com/axibase/webtest/cases/CreatePortalTest.java]: Travis was run on wrong branch, now it's fixed. Let's merge this PR

**Pull request merged**